cvat-ui/src/components/annotation-page/annotation-page.tsx

```diff
@@ -79,6 +80,11 @@ export default function AnnotationPageComponent(props: Props): JSX.Element {
             <Layout.Header className='cvat-annotation-header'>
                 <AnnotationTopBarContainer />
             </Layout.Header>
+            {workspace === Workspace.STANDARD3D && (
+                <Layout.Content style={{ height: '100%' }}>
```

Comment:
**style** doesn't contains any dynamic variables, so, please use the css class in places like this